ci-opal.yml

```diff
@@ -42,6 +43,18 @@ builders:
   - "{{ .Image }}"
   - /sbin/init
   commit: true
+- type: openstack
+  name: openstack
+  flavor: m1.medium-travis-ci
+  image_name: "{{ user `image_name` }}"
+  ssh_username: ubuntu
+  networks:
+  <% ENV['OS_NETWORKS'].to_s.split(',').map(&:strip).each do |network| %>
+  - <%= network %>
+  <% end %>
+  source_image_name: "{{ user `openstack_source_image_name` }}"
+  ssh_keypair_name: osu_openstack_key
+  ssh_private_key_file: /root/osu_openstack_key
```

Comment:
This `ssh_private_key_file` value does not work for me locally, which I'm assuming is because it is an absolute path.  Can this value be injected via an environment variable instead?